### PR TITLE
[EL] Add HassNevermind 

### PR DESCRIPTION
**Pull request by apo-mak**
Add Greek support for a future HA intent: HassNevermind (see https://github.com/home-assistant/intents/pull/1549).

### CODE REVIEW

File: sentences/el/homeassistant_HassNevermind.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: el
+intents:
+  HassNevermind:
+    data:
+      - sentences:
+          - "άστο"
+          - "ξέχνατο"
```
Comment (dsimop) [inline sentences/el/homeassistant_HassNevermind.yaml:7]: Please apply this minor fix: "ξέχνα το"

File: tests/el/homeassistant_HassNevermind.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: el
+tests:
+  - sentences:
+      - "άστο"
+      - "ξέχνατο"
```
Comment (dsimop) [inline tests/el/homeassistant_HassNevermind.yaml:5]: Please apply this minor fix: "ξέχνα το"

**Review by dsimop — CHANGES_REQUESTED** [review]
Please apply this minor fix: "ξέχνα το" to the commented lines.

**Comment by apo-mak** [comment]
> Please apply this minor fix: "ξέχνα το" to the commented lines.

I tried it with google TTS and it does a strange pause when I split the two.
I do not know how the STT would prefer to have it, but we also tend to say it almost like one word.
Should we put both in for better accuracy?


(I can make a new pull after this , with out the split.
This will help reverting it if necessary later.)

**Review by dsimop — APPROVED** [review]

**Comment by dsimop** [comment]
Hi @apo-mak, sure you can add it as one word as well.

**Pull request merged**